tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -247,6 +247,80 @@ static void test_advx_main(void)
 	}
 	printk("success.\n");
 
+	k_sleep(K_MSEC(400));
+
+	printk("Removing adv aux set that's created and disabled ...");
+	err = ll_adv_aux_set_remove(handle);
+	if (err) {
+		printk("err = %d\n", err);
```

Comment:
All the printk's are redundant, as err is printed in line 330.

Reply:
Oh, sorry, I added those during my testing. Will remove.

Reply:
Fixed.